packages/page-template-modal/package.json

```diff
@@ -28,7 +28,7 @@
 		"src"
 	],
 	"dependencies": {
-		"@automattic/js-utils": "^1.0.0",
+		"@automattic/js-utils": "^0.1.0",
```

Comment:
Note: we should remove the package from here but I'd suggest a follow up /cc @tyxla

Reply:
Follow-up sounds good 👍  I'll be getting rid of `isNullish` in another PR.